oci/pack.go

```diff
@@ -272,6 +272,16 @@ func NewPackageFromTarget(ctx context.Context, targ target.Target, opts ...packm
 
 	ocipack.manifest.SetOS(ctx, ocipack.Platform().Name())
 	ocipack.manifest.SetArchitecture(ctx, ocipack.Architecture().Name())
+	ocipack.manifest.SetEnv(ctx, popts.Env())
+	for _, env := range ocipack.manifest.config.Config.Env {
+		split := strings.SplitN(env, "=", 2)
+		k := split[0]
+		v := ""
+		if len(split) > 1 {
+			v = split[1]
+		}
```

Comment:
Sorry, I was interrupted and was not able to do a full pass in a single sitting.

This can be reduced to:
```go
k, v, _ := strings.Cut(env, "=")
```